### PR TITLE
Fix incorrect words and formats in image

### PR DESCRIPTION
**Pull request by yuexiao-wang**
**- What I did**
1. Fix incorrect words such as "Invalid" and "Exected"
2. Error information should begin with lower-case 
3. Supplement some information

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

### CODE REVIEW

File: image/fs_test.go [hunk 1]
```diff
@@ -261,7 +261,7 @@ func testGetSet(t *testing.T, store StoreBackend) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	// skipping use of digest pkg because its used by the implementation
+	// skipping use of digest pkg because its is used by the implementation
```
Comment (allencloud) [inline image/fs_test.go]: its -> it ?

Comment (yuexiao-wang) [inline image/fs_test.go]: Sorry,  It will be applied to this PR.
Thanks a lot.

**Comment by yuexiao-wang** [comment]
@thaJeztah PTAL
Combine #29351 into this PR.
Thanks

**Comment by yuexiao-wang** [comment]
@thaJeztah @vdemeester PTAL

File: image/fs_test.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func TestFSGetInvalidData(t *testing.T) {
 
 	_, err = fs.Get(id)
 	if err == nil {
-		t.Fatal("Expected get to fail after data modification.")
+		t.Fatal("expected to get failed after data modification.")
```
Comment (vdemeester) [inline image/fs_test.go]: Pretty sure the initial message is better (we expect the `get` method to fail)

File: image/fs_test.go [hunk 3]
```diff
@@ -75,7 +75,7 @@ func TestFSInvalidSet(t *testing.T) {
 
 	_, err = fs.Set([]byte("foobar"))
 	if err == nil {
-		t.Fatal("Expecting error from invalid filesystem data.")
+		t.Fatal("expecting error from invalid filesystem data.")
```
Comment (vdemeester) [inline image/fs_test.go]: s/expecting/expected

**Review by vdemeester — CHANGES_REQUESTED** [review]

**Comment by yuexiao-wang** [comment]
@vdemeester PTAL
Done
Thanks

File: image/fs_test.go [hunk 4]
```diff
@@ -208,7 +208,7 @@ func TestFSWalker(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	testWalker(t, fs)
+	testWalk(t, fs)
```
Comment (thaJeztah) [inline image/fs_test.go]: I'm not sure this is correct; it's testing a _walker_ (that "walks")? In any case, I'm not sure we gain much by renaming these

Comment (yuexiao-wang) [inline image/fs_test.go]: @thaJeztah PTAL
OK and reverted them.

File: image/image.go [hunk 5]
```diff
@@ -29,11 +29,11 @@ func IDFromDigest(digest digest.Digest) ID {
 
 // V1Image stores the V1 image configuration.
 type V1Image struct {
-	// ID a unique 64 character identifier of the image
+	// ID is a unique 64 character identifier of the image
 	ID string `json:"id,omitempty"`
 	// Parent id of the image
```
Comment (thaJeztah) [inline image/image.go]: Looking at this again, I think I know why these comments were inconsistent. I think the original intent of these was to write;

```
ID - a unique 64 character identifier of the image
Comment - user added comment
```

Which doesn't follow GoDoc conventions. _However_, now that _some_ follow those conventions, and some don't, we should be careful;

```
Parent id of the image
```

could now be interpreted as

```
Parent - ID of the image
```

Which isn't correct. I suggest (but further suggestions welcome);

```
Parent is the ID of the parent image
Comment is the commit message that was set when committing the image
Created is the timestamp at which the image was created
DockerVersion specifies the version of Docker that was used to build the image
Author is the name of the author that was specified when committing the image
```

File: image/tarexport/tarexport.go [hunk 6]
```diff
@@ -36,7 +36,7 @@ type LogImageEvent interface {
 	LogImageEvent(imageID, refName, action string)
 }
 
-// NewTarExporter returns new ImageExporter for tar packages
+// NewTarExporter returns new Exporter in image for tar packages
```
Comment (thaJeztah) [inline image/tarexport/tarexport.go]: not sure we need "in image" 

```
NewTarExporter returns a new Exporter for saving images as tar packages
```

@vdemeester does that look correct? ^

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: image/tarexport/tarexport.go (hunk 6 above)
Comment (vdemeester) [inline image/tarexport/tarexport.go]: I would say yes :angel:

**Comment by yuexiao-wang** [comment]
@thaJeztah @vdemeester PTAL
thanks

File: image/image.go [hunk 7]
```diff
@@ -112,21 +112,21 @@ func (img *Image) MarshalJSON() ([]byte, error) {
...
+	// Created is timestamp when image was created
 	Created time.Time `json:"created"`
-	// Author of the build point
+	// Author is the name of the author that was specified when building the image
 	Author string `json:"author,omitempty"`
-	// CreatedBy keeps the Dockerfile command used while building image.
+	// CreatedBy keeps the Dockerfile command used while building the image
 	CreatedBy string `json:"created_by,omitempty"`
-	// Comment is custom message set by the user when creating the image.
+	// Comment is custom message set by user when creating the image.
```
Comment (thaJeztah) [inline image/image.go]: can use the same here as above;

```
Comment is the commit message that was set when committing the image
```

File: image/image.go [hunk 8]
```diff
@@ -112,21 +112,21 @@ func (img *Image) MarshalJSON() ([]byte, error) {
 
 // History stores build commands that were used to create an image
 type History struct {
-	// Created timestamp for build point
+	// Created is timestamp when image was created
```
Comment (thaJeztah) [inline image/image.go]:
```
Created is the timestamp at which the image was created
```

File: image/image.go [hunk 9]
```diff
@@ -29,21 +29,21 @@ func IDFromDigest(digest digest.Digest) ID {
 
 // V1Image stores the V1 image configuration.
 type V1Image struct {
-	// ID a unique 64 character identifier of the image
+	// ID is a unique 64 character identifier of the image
 	ID string `json:"id,omitempty"`
-	// Parent id of the image
+	// Parent is the id of the image
```
Comment (thaJeztah) [inline image/image.go]: You missed "parent" here;

```
Parent is the ID of the parent image
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Some issues remaining, see my comments inline

**Comment by yuexiao-wang** [comment]
@thaJeztah PTAL
Thanks!

File: image/image.go [hunk 10]
```diff
@@ -112,21 +112,21 @@ func (img *Image) MarshalJSON() ([]byte, error) {
 
 // History stores build commands that were used to create an image
 type History struct {
-	// Created timestamp for build point
+	// Created is the timestamp at which the image was created
 	Created time.Time `json:"created"`
-	// Author of the build point
+	// Author is the name of the author that was specified when building the image
```
Comment (yuexiao-wang) [inline image/image.go]: @thaJeztah 
It uses "building the image" here , right?

Comment (thaJeztah) [inline image/image.go]: Should probably be `committing` as well here, given that this value is set by the `--author` flag on `docker commit` https://docs.docker.com/engine/reference/commandline/commit/

**Comment by yuexiao-wang** [comment]
@thaJeztah PTAL
It has been done.
Thanks

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
@vdemeester PTAL

**Review by vdemeester — APPROVED** [review]
LGTM :frog:

**Pull request merged**